applications/nrf_desktop/configuration/nrf52810dmouse_nrf52810/prj.conf

```diff
@@ -108,13 +108,6 @@ CONFIG_BT_CTLR_RL_SIZE=2
 CONFIG_BT_GATT_UUID16_POOL_SIZE=27
 CONFIG_BT_GATT_CHRC_POOL_SIZE=7
```

Comment:
Consider adding defaults for the GATT Pool module as the pool elements seem only to be used by the HID service. This configuration is directly affected by the chosen HID report configuration

Reply:
That would be aligned together with UUID16 pool size (in a separate PR)